src/ARToolkitNFT.ts

```diff
@@ -272,7 +269,7 @@ export default class ARToolkitNFT {
       onError2(errorNumber);
     };
 
-    let Ids: any = [];
+    let Ids: [{id: number}];
```

Comment:
This is correc but a silent error, when the marker is loaded you will receive this kind of error:
```
aa1e448a-43ef-43be-881e-d5c11c836675:1 Error in loading marker on Worker TypeError: Cannot read properties of undefined (reading 'push')
    at aa1e448a-43ef-43be-881e-d5c11c836675:1:748671
    at Array.forEach (<anonymous>)
    at A.value (aa1e448a-43ef-43be-881e-d5c11c836675:1:748283)
    at A.<anonymous> (aa1e448a-43ef-43be-881e-d5c11c836675:1:755365)
    at G (aa1e448a-43ef-43be-881e-d5c11c836675:1:928)
    at Generator.<anonymous> (aa1e448a-43ef-43be-881e-d5c11c836675:1:2283)
    at Generator.next (aa1e448a-43ef-43be-881e-d5c11c836675:1:1291)
    at A (aa1e448a-43ef-43be-881e-d5c11c836675:1:8095)
    at i (aa1e448a-43ef-43be-881e-d5c11c836675:1:8298)
    at aa1e448a-43ef-43be-881e-d5c11c836675:1:8357
(anonymous) @ aa1e448a-43ef-43be-881e-d5c11c836675:1
Promise.catch (async)
(anonymous) @ aa1e448a-43ef-43be-881e-d5c11c836675:1
Promise.then (async)
D @ aa1e448a-43ef-43be-881e-d5c11c836675:1
C.onmessage @ aa1e448a-43ef-43be-881e-d5c11c836675:1

```